api/tests/core/offers/test_offer_metadata.py

```diff
@@ -139,6 +139,36 @@ def should_have_a_location(self):
                 },
             }
 
+        def should_have_an_url(self):
+            offer = offers_factories.EventOfferFactory(url="https://passculture.app/offre/72180399")
+
+            offers_factories.StockFactory(offer=offer)
+
+            metadata = get_metadata_from_offer(offer)
+
+            assert metadata["offers"]["url"] == "https://passculture.app/offre/72180399"
+
+        def should_have_online_event_attendance_mode(self):
+            offer = offers_factories.EventOfferFactory(url="https://passculture.app/offre/72180399")
+
+            metadata = get_metadata_from_offer(offer)
+
+            assert metadata["eventAttendanceMode"] == "OnlineEventAttendanceMode"
+
+        def should_have_offline_event_attendance_mode(self):
+            offer = offers_factories.EventOfferFactory()
+
+            metadata = get_metadata_from_offer(offer)
+
+            assert metadata["eventAttendanceMode"] == "OfflineEventAttendanceMode"
+
+        def should_have_valid_from_date(self):
+            offer = offers_factories.EventOfferFactory(extraData={"releaseDate": "2000-01-01T18:09:09Z"})
```

Comment:
```suggestion
            offer = offers_factories.EventOfferFactory(extraData={"releaseDate": "2000-01-01"})
```

la date de parution a l'air d'être une date et non un datetime :

```
➜ rg "releaseDate\": \""
tests/local_providers/allocine_stocks_functions_test.py
163:            "releaseDate": "2021-05-19",

tests/local_providers/allocine_stocks_test.py
238:            "releaseDate": "2001-10-03",
263:            "releaseDate": "2001-10-03",
614:            "releaseDate": "2001-10-03",
629:            "releaseDate": "2001-10-03",
```